recipes/cudatoolkit-dev/cudatoolkit-dev-post-install.py

```diff
@@ -0,0 +1,388 @@
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. """
+
+from __future__ import print_function
+
+import json
+import os
+import shutil
+import stat
+import subprocess
+import sys
+import urllib.parse as urlparse
+from pathlib import Path
+
+from conda.exports import download, hashsum_file
+
+
+def set_chmod(file_name):
+    # Do a simple chmod +x for a file within python
+    st = os.stat(file_name)
+    os.chmod(file_name, st.st_mode | stat.S_IXOTH)
+
+
+def create_dir(new_dir):
+    try:
+        os.makedirs(new_dir)
+    except FileExistsError:
+        pass
+
+
+def copy_files(src, dst):
+    try:
+        if (os.path.isfile(src)):
+            set_chmod(src)
+            shutil.copy(src, dst)
+    except FileExistsError:
+        pass
+
+
+class Extractor(object):
+    """Extractor base class, platform specific extractors should inherit
+    from this class.
+    """
+    libdir = {'linux': 'lib',
+              'osx': 'lib',
+              'windows': 'DLLs', }
+
+    def __init__(self, cudatoolkit_config, platform_config):
+        """Initialise an instance:
+        Arguments:
+          cudatoolkit_config: the configuration for CUDA
+          platform_config - the configuration for this platform
+        """
+        self.cu_name = cudatoolkit_config['name']
+        self.cu_version = cudatoolkit_config['version']
+        self.md5_url = cudatoolkit_config["md5_url"]
+        self.base_url = cudatoolkit_config["base_url"]
+        self.patch_url_text = cudatoolkit_config["patch_url_ext"]
+        self.installers_url_ext = cudatoolkit_config["installers_url_ext"]
+        self.cu_blob = platform_config['blob']
+        self.conda_prefix = os.environ.get('CONDA_PREFIX')
+        self.prefix = os.environ["PREFIX"]
+        self.src_dir = Path(self.conda_prefix) / 'pkgs' / self.cu_name
+        try:
+            os.makedirs(self.src_dir)
+
+        except FileExistsError:
+            pass
+
+        self.symlinks = getplatform() == "linux"
+        self.debug_install_path = os.environ.get('DEBUG_INSTALLER_PATH')
+
+    def create_activate_and_deactivate_scripts(self):
+        activate_dir_path = Path(self.conda_prefix) / \
+            'etc' / 'conda' / 'activate.d'
+        deactivate_dir_path = Path(
+            self.conda_prefix) / 'etc' / 'conda' / 'deactivate.d'
+
+        try:
+            os.makedirs(activate_dir_path)
+            os.makedirs(deactivate_dir_path)
```

Comment:
These should be in separate try-except blocks.  But rather than having them in try-blocks at all, `makedirs()` has an `exist_ok` argument https://docs.python.org/3/library/os.html#os.makedirs So this should probably just be:

```python
os.makedirs(activate_dir_path, exist_ok=True)
os.makedirs(deactivate_dir_path, exist_ok=True)
``` 

without the try block